templates/enhancer-config-pod.json

```diff
@@ -26,4 +26,4 @@
       "@type": "EnhancementHandlerVocabulary"
     }
   ]
-}
+}
```

Comment:
There's a missing newline at the end of this file. Could you add one? Then git will just see this as a file rename, without differences.